### PR TITLE
V10 cjs compatibility.

### PR DESCRIPTION
**Pull request by tripodsgames**
PR to transpile ESM to CommonJS using esbuild.

https://github.com/RIAEvangelist/node-ipc/discussions/206#discussioncomment-966822

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ A great solution for complex multiprocess **Neural Networking** in Node.JS
 
 **npm install node-ipc**
 
+#### for node <v14
+
+set version to `<10.0.0` in your `package.json` for the node-ipc.
+
```
Comment (RIAEvangelist) [inline README.md:12]: Cany you provide a very basic example of this in a code block here. Some users may be new enough that they don't know how. Just the single line example of @10 or whatever would do.

**Review by RIAEvangelist — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (RIAEvangelist) [reply README.md:12]: also do we need any documentation for the `npm run prepare` script, or will that automatically be run by the transpilers?

Comment (tripodsgames) [reply README.md:12]: `prepare` runs automatically, It's called by npm.
https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

**Comment by RIAEvangelist** [comment]
@tripodsgames can you explain or address testing failures?

**Comment by tripodsgames** [comment]
Fixed! @RIAEvangelist

**Comment by RIAEvangelist** [comment]
published

**Pull request merged**